src/frontend/index.js

```diff
@@ -10,41 +10,34 @@ requirejs.config({
         { name: 'mod_data_structures', location: '../backend/data_structures', main: 'index'},
         { name: 'mod_parser', location: '../backend/parser', main: 'index'},
         { name: 'cm', location: '../bower_components/codemirror', main: 'codemirror'},
+        { name: 'jquery', location: '../bower_components/jquery/dist', main: 'jquery.min'},
         { name: 'backend', location: '../backend', main: 'index'},
         { name: 'code_input', location: 'code_input', main: 'index'},
+        { name: 'console', location: 'console', main: 'index'},
         { name: 'visualization', location: 'visualization', main: 'index'},
+        { name: 'interface', location: 'interface', main: 'index'},
+        { name: 'visualization', location: 'visualization', main: 'index'},
```

Comment:
All modules will probably have more code, and at this stage changing it will took too much time and is completly not necessary (unless I missed something)